src/utils.ts

```diff
@@ -145,6 +153,28 @@ export const getEmailContent = (
           We do offer the possibility for you to apply again next time if you meet the eligibility criteria. We invite you to stay engaged with us by attending our events, reaching out to our admissions team, and taking advantage of any opportunities to connect with our current students and alumni.<br /><br />
           Thank you again for considering our program and for the time you invested in your application. We wish you all the best in your future endeavours.`
         }
+      case ApplicationStatus.COMPLETED: {
+        const pdfFileName = await generateCertificate(
+          name,
+          './src/certificates/certificate_template.pdf',
+          `./src/certificates/${name}_certificate.pdf`
```

Comment:
Shall we use a unique id here? There might be a chance of having the same name. Also you need to update the certificate_id field of the mentee with file path as well.

 
```suggestion
          `./src/certificates/${name}_certificate.pdf`
```

Reply:
okay sure

Reply:
![image](https://github.com/user-attachments/assets/dbe79847-36a9-456e-92cc-021d8a8b3698)
here I used uuid library to generate unique values